packages/react/src/components/Storage/File-Uploader/hooks/useFileUploader/types.ts

```diff
@@ -0,0 +1,19 @@
+export type SetFileType = File[];
+
+export interface DragActionHandlers {
+  onDragStart: (e: React.DragEvent<HTMLDivElement>) => void;
```

Comment:
nit: 
```suggestion
  onDragStart: (event: React.DragEvent<HTMLDivElement>) => void;
```

so that its name is more explicit whenever customer sees this signature through vscode.

Reply:
Sure